client/package.json

```diff
@@ -15,7 +16,7 @@
   "scripts": {
     "build-css": "node-sass-chokidar src/ -o src/",
     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
-    "start": "react-scripts start",
+    "start": "npm run build-css && react-scripts start",
```

Comment:
Not sure that's necessary. We should be running watch-css all the time anyway. Unless it's to ensure the css is built before starting the app? My concern is it might be a little too easy to forget to run watch-css and then waste time wondering why the changes to the scss are not applied.

Reply:
Exactly, you will always have an up to date version of your scss every time you run the server. I do not see this as a negative thing.